modules/programs/aerospace.nix

```diff
@@ -0,0 +1,231 @@
+{ config, lib, pkgs, ... }:
+let
+  inherit (lib) mkOption types;
+  cfg = config.programs.aerospace;
+
+  tomlFormat = pkgs.formats.toml { };
+
+  filterAttrsRecursive = pred: set:
```

Comment:
Please add a comment explaining why you didn't use the `filterAttrsRecursive` function from Nixpkgs. Ideally perhaps use a different function name.